doc/api/http.md

```diff
@@ -132,6 +137,9 @@ added: v0.3.4
   * `maxFreeSockets` {number} Maximum number of sockets to leave open
     in a free state. Only relevant if `keepAlive` is set to `true`.
     **Default:** `256`.
+  * `scheduling` {string} Scheduling strategy to apply when picking
+    the next free socket to use. It can be `'fifo'` or `'lifo'`.
+    **Default:** `'fifo'`
```

Comment:
Would be ideal to have `fifo` and `lifo` briefly explained as not all readers may be familiar with the terms. Might also be good to briefly explain why one would stray from the default.

Reply:
Thank you for the suggestion! Let me know what you think :)